index.html

```diff
@@ -206,8 +199,27 @@ <h1 class="title">Seminar Hall Booking</h1>
                     </div> <!-- item -->
                   </div> <!-- .form-item -->
                 </div>  <!-- .form-item-container -->
-
-                <div id="status-table-container" class="form-item-container">
```

Comment:
Be careful while adding new items. You seemed to have removed the id of the table container. Bring it back.